src/actions/index.js

```diff
@@ -5,17 +5,17 @@ export const RECEIVE_FORECAST = 'RECEIVE_FORECAST';
 export const REQUEST_FORECAST = 'REQUEST_FORECAST';
 const API_URL = 'http://api.openweathermap.org/data/2.5/weather?zip=94040,us&appid=b248d23bebc3f14165312ce58845f0c4'
 
-export function fetchWeatherByZip(zip) {
-  return dispatch => {
-    dispatch({ type: REQUEST_FORECAST });
-    return fetch(API_URL.replace(/zip/, zip))
-      .then(req => req.json())
-      .then(json => dispatch({
-        type: RECEIVE_FORECAST,
-        json
-      }));
-  };
-}
+// export function fetchWeatherByZip(zip) {
```

Comment:
Commented out code is fine when you're working on a feature, but should be removed before merging into master. 😃